website/src/components/wrapper/wrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Children, cloneElement } from 'react'
+import Playground from 'components/playground'
+import PropsTable from 'components/props-table'
+
+export type WrapperProps = {
+  children: React.ReactNode
+}
+
+export function Wrapper({ children }: WrapperProps) {
```

Comment:
This component is not clear and easy to understand.

We should add some comments to this one.